### ISSUE TITLE
Let's make this project active again!

### ISSUE THREAD

**Issue by fafato1** [npruehs/ue4-rts#208]
It's such a awesome project! We can modify to use the most news Unreal systems, like Smart Object, Mover (When goes stable) and other things!

**Comment by npruehs** [comment]
Hey, thanks for the nice words :) Unfortunately, I've got two kids and two jobs now - one of which is at Frost Giant Studios, which somehow might create a conflict of interest... I'm happy to pick this up at a later time though, and provide any necessary guidance in the meantime.

**Comment by fafato1** [comment]
YOU'RE WORKINT AT FROST GIANT??? This is awesome! I'm your fan now! It's ok, i'll try to help with upgrades and updates! I will be happy with your support and guidance

**Comment by Commyno** [comment]
I agree, this project is spectacular and it's a real shame that it remains here to gather dust. A great job that needs support. Npruehs, two kids is the real challange :-P!
